rq/group.py

```diff
@@ -119,6 +118,7 @@ def clean_registries(cls, connection: 'Redis'):
         groups = connection.smembers(Group.REDIS_GROUP_KEY)
```

Comment:
Let's expose a method `Group.all(connection)`, to make it symmetric with `Worker.all()`

Reply:
Done